excellalabs/planning-poker-server
# Session creation

## Description
Added classes and mutation for ability to persist a session collection to mongo

## How to Test
Start server, go to `localhost:3001/graphql`, persist
```graphql
mutation {
  createSession { 
    _id
    sessionId
  }
}
```
and see uniquely generated `_id` and the hardcoded `sessionId` = '1234'
